community/openebs/templates/deployment-maya-apiserver.yaml

```diff
@@ -49,9 +54,9 @@ spec:
         # configured as a part of openebs installation.
         # If "true" a default cstor sparse pool will be configured, if "false" it will not be configured.
         - name: OPENEBS_IO_INSTALL_DEFAULT_CSTOR_SPARSE_POOL
-          value: "{{ .Values.ndm.sparse.enabled }}"
-{{- end }}
-{{- end }}

```

Comment:
We have to close the if conditional loop.

Reply:
Done.